sympy/core/mul.py

```diff
@@ -295,12 +295,8 @@ def flatten(cls, seq):
                 continue
 
             elif o.is_commutative:
-                #      e
-                # o = b
                 b, e = o.as_base_exp()
 
-                #  y
-                # 3
```

Comment:
Don't remove these comments.

Reply:
It's giving an example of what the next if-block is handling: a number raised to some power.